catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveAllCommandTest.java

```diff
@@ -72,35 +108,79 @@ public void testBadBatchSize() throws Exception {
     }
 
     /**
+     * Tests that if a current batch to be removed is less than the batch size,
+     * but items still remain to be deleted, command will not terminate prematurely.
+     *
+     * Catalog Fra
+     * <p>
+     * Cumulative results by each batch:
+     * First:       50
+     * Second:      100
+     * Third:       150
+     * Fourth:      200 (stops at 200)
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testDeleteFewerThanBatchSizeExpectMultipleCatalogCalls() throws Exception {
+
+        resultsPerCatalogQuery = 50;
```

Comment:
✏️ I would rather we not use such _pretty_ numbers, prone to fencepost errors. I think this test would be more clear if you had numbers that did not evenly divide into one another.